bookkeeper-server/src/test/java/org/apache/bookkeeper/client/TestLedgerFragmentReplication.java

```diff
@@ -111,7 +111,7 @@ public void testReplicateLFShouldCopyFailedBookieFragmentsToTargetBookie()
         // 0-9 entries should be copy to new bookie
 
         for (LedgerFragment lf : result) {
-            admin.replicateLedgerFragment(lh, lf);
+            admin.replicateLedgerFragment(lh, lf, null);
```

Comment:
Isn't it better to always pass a non null value?
It will make it clearer the flow of that variable where it is used, we can prevent bugs

Reply:
@eolivelli even I've mixed opinions about this null thing. In retrospect, instead of passing ConcurrentSet it would be better if I pass function (callback) to the bottom of stack and call this function incase of read failure. So callers can decide what they want to have in their callback logic. Here instead of passing null, it can pass no-op function.

Reply:
Sounds good